contents/swift/welcome-to-swift.md

```diff
@@ -312,7 +353,26 @@ Reference cycles are bad, because they cause memory leaks. Even though A and B a
 
 Here is an [article](https://krakendev.io/blog/weak-and-unowned-references-in-swift) with greater in-depth explanation and examples.
 
-## CocoaPods
+### Benefit: Reuse Existing Code
+
+[CocoaPods](https://guides.cocoapods.org/using/getting-started.html) is a dependency manager for Swift and Objective-C Cocoa projects which has over 58 thousand libraries and is used in over 3 million apps. Instead of reinventing the wheel, you can check this out to obtain code that helps resolve common issues. If you felt you have done something new with Swift, you can also make your code into a library with CocoaPods.
+
+## How to Get Started
+
+A Macbook is required for Swift development, but iPhone or iPad are not. The Swift IDE `X-Code` has built-in simulators for all mobile devices.
+
+If you have not learnt any other programming languages before, this [Game App](https://www.apple.com/swift/playgrounds/) could be a good choice to learn swift as well as programming.
+
+If you have been familiar with some programming language, 
+reading the [Language Guide in the official documentation](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/) is recommended, since it explains everything quite clearly, albeit quite verbose. In addition, links that further explain the cool features are put the the end of each feature.
```

Comment:
If you are already familiar with another programming language, reading [Language Guide in the official documentation (https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/) is recommended. The documentation should give you a comprehensive overview of the functionality within Swift. 

With regards to the cool features links part, I skimmed through the documentation, what links are you referring to?

Reply:
Oh, I refer to the links at the end of features in this book chapter. (i.e. the links at the end of Protocol Orienting Programming section) Probably I delete this sentence since it could be misleading